scissors-sample/src/main/java/com/lyft/android/scissorssample/CropResultActivity.java

```diff
@@ -42,10 +51,17 @@ protected void onCreate(Bundle savedInstanceState) {
         String filePath = getIntent().getStringExtra(EXTRA_FILE_PATH);
         File imageFile = new File(filePath);
 
-        Picasso.with(this)
-                .load(imageFile)
-                .memoryPolicy(MemoryPolicy.NO_CACHE, MemoryPolicy.NO_STORE)
-                .into(resultView);
+        DisplayImageOptions options = new DisplayImageOptions.Builder()
+                .cacheInMemory(false)
+                .cacheOnDisk(false)
+                .build();
+        ImageLoader.getInstance().displayImage("file://" + filePath, resultView, options);
```

Comment:
As mentioned before comment it out leaving picasso only e.g.

`// Or Android-Universal-Image-Loader`